Localization/StringsConvertor/input/ar.lproj/app.json

```diff
@@ -376,7 +382,11 @@
                 "video": "مقطع مرئي",
                 "attachment_broken": "هذا ال%s مُعطَّل\nويتعذَّرُ رفعُه إلى ماستودون.",
                 "description_photo": "صِف الصورة للمَكفوفين...",
-                "description_video": "صِف المقطع المرئي للمَكفوفين..."
+                "description_video": "صِف المقطع المرئي للمَكفوفين...",
+                "load_failed": "Load Failed",
+                "upload_failed": "Upload Failed",
+                "can_not_recognize_this_media_attachment": "Can not regonize this media attachment",
```

Comment:
There's a typo here:

`Can not regonize […]` -> `Can not recognize […]`

Or maybe rephrase like `Failed to recognize […]`